worklets/idlharness.https.any.js

```diff
@@ -0,0 +1,10 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+// https://drafts.css-houdini.org/worklets/
+
+'use strict';
+
+idl_test(
+  ['worklets']
```

Comment:
This didn't work, see failures in https://wpt.fyi/results/worklets/idlharness.https.any.html?sha=4b6ec0edaf&label=pr_head. I think you'll need to supply another empty array for dependencies.